modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/view_content_dashboard_item_types.jsp

```diff
@@ -39,14 +39,13 @@ ContentDashboardItemTypeItemSelectorViewDisplayContext contentDashboardItemTypeI
 			<%
 			InfoItemReference infoItemReference = contentDashboardItemType.getInfoItemReference();
 
-			row.setData(
-				HashMapBuilder.<String, Object>put(
-					"className", infoItemReference.getClassName()
-				).put(
-					"classPK", infoItemReference.getClassPK()
-				).put(
-					"title", contentDashboardItemType.getFullLabel(locale)
-				).build());
+			row.setPrimaryKey(
+				HtmlUtil.toInputSafe(
+					JSONUtil.put(
+						"className", infoItemReference.getClassName()
```

Comment:
What about `infoItemIdentifier`?

```
infoItemReference.getInfoItemIdentifier()
```

I saw that this data contains the classPK and className but I don't know if it fits our case.

Reply:
Hey @dgarciasarai!!!

We need the className and classPK, this info is only provided if the infoItemIdentifer is of ClassPKInfoItemIdentifier type (this logic is encapsulated in infoItemReference.getClassPK()), otherwise we should have a instanceof everytime registered by the portal, to be able to obtain the className and classPK.

To keep the same logic we use in 7.3, we have decided to use the same way of obtain the className and classPK that we use before.

Thanks!!!